ctapipe/analysis/Sensitivity.py

```diff
@@ -0,0 +1,790 @@
+from itertools import chain
+import astropy.units as u
+from astropy.table import Table
+from scipy.optimize import minimize
+import scipy.integrate as integrate
+
+import numpy as np
+
+from matplotlib import pyplot as plt
+
+
+# tau, the superior circle number
+np.tau = 2*np.pi
+
+__all__ = ["SensitivityPointSource"]
+
+
+def convert_astropy_array(arr, unit=None):
```

Comment:
This function is really a one-liner and only called once. Suggest to remove it.
(I can have a closer look and suggest how to write the original function where you call this if there's some problem that makes the code there hard to read).